File: basic/src/main/js/Infringement.js
import React from 'react';
import { Link } from 'react-router';

export default class Infringement extends React.Component{
	render() {
		return (
			<tr>
				
				<td>{this.props.infringement.category}</td>
				<td>{this.props.infringement.noOfPns}</td>
				<td>{this.props.infringement.faceValue}</td>
			</tr>
		)
	}
}

File: basic/src/main/java/com/sydneymetro/infringements/Infringement.java

package com.sydneymetro.infringements;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

import lombok.Data;


@Data
@Entity
public class Infringement {

	private @Id @GeneratedValue Long id;
	private int year;
	private String category;
	private long noOfPns;
	private long faceValue;

	private Infringement() {}

	public Infringement(int year,String category,long noOfPns,long faceValue) {
		this.year = year;
		this.category = category;
		this.noOfPns = noOfPns;
		this.faceValue=faceValue;
	}
}
